### PR TITLE
Migrate kafka.

### PR DESCRIPTION
**Pull request by jacobcrawford**
**Description of your changes:**
Migrate kafka to mono repo

Checklist:

* [x] I have bumped the chart version
* [ ] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [ ] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
* [x] I have created the necessary tests in the chart, if they are possible/necessary
* [x] I have squashed commits if necessary

### CODE REVIEW

File: charts/kafka/chart/Chart.yaml [hunk 1]
```diff
@@ -2,11 +2,12 @@ apiVersion: v2
 name: kafka
 description: A Helm chart for Kubernetes
 type: application
-version: 0.2.2
+version: v0.3.1
```
Comment (skovbakke) [inline charts/kafka/chart/Chart.yaml:5]:
```suggestion
version: 0.3.1
```

**Review by skovbakke — COMMENTED** [review]
The linting fails.

File: charts/kafka/chart/Chart.yaml [hunk 2]
```diff
@@ -2,11 +2,12 @@ apiVersion: v2
 name: kafka
 description: A Helm chart for Kubernetes
 type: application
-version: 0.2.2
+version: 0.3.1
+appVersion: "1.16.0"
```
Comment (jaxels10) [inline charts/kafka/chart/Chart.yaml:6]: Remove pl0x

**Review by jaxels10 — CHANGES_REQUESTED** [review]
1 comment

**Review by jaxels10 — APPROVED** [review]
This looks okay to me. Jacob is vouching that it is okay that the helm install fails

**Pull request merged**